core/src/main/java/com/uber/m3/tally/Reportable.java

```diff
@@ -0,0 +1,9 @@
+package com.uber.m3.tally;
+
+import com.uber.m3.util.ImmutableMap;
+
+interface Reportable {
```

Comment:
nit: comment the intent of the interface

Reply:
Added